test/CollectionTest.cc

```diff
@@ -163,56 +164,51 @@ class CollectionTest : public CBLTest {
         ExpectingExceptions x;
         
         CBLError error = {};
-        CHECK(!CBLDatabase_DefaultScope(db, &error));
+        CHECK(!CBLDatabase_DefaultScope(database, &error));
         CheckNotOpenError(error);
         
         error = {};
-        CHECK(!CBLDatabase_DefaultCollection(db, &error));
+        CHECK(!CBLDatabase_DefaultCollection(database, &error));
         CheckNotOpenError(error);
         
         error = {};
-        CHECK(!CBLDatabase_ScopeNames(db, &error));
+        CHECK(!CBLDatabase_ScopeNames(database, &error));
         CheckNotOpenError(error);
         
         error = {};
-        CHECK(!CBLDatabase_CollectionNames(db, "_default"_sl,  &error));
+        CHECK(!CBLDatabase_CollectionNames(database, "_default"_sl,  &error));
         CheckNotOpenError(error);
         
         error = {};
-        CHECK(!CBLDatabase_Collection(db, "_default"_sl, "_default"_sl, &error));
+        CHECK(!CBLDatabase_Collection(database, "_default"_sl, "_default"_sl, &error));
         CheckNotOpenError(error);
         
         error = {};
-        CHECK(!CBLDatabase_Scope(db, "_default"_sl, &error));
+        CHECK(!CBLDatabase_Scope(database, "_default"_sl, &error));
         CheckNotOpenError(error);
     }
 };
 
 #define NOT_DELETE_DEFAULT_COLLECTION
 
 TEST_CASE_METHOD(CollectionTest, "Default Collection", "[Collection]") {
-    CBLError error = {};
-    CBLCollection* col = CBLDatabase_DefaultCollection(db, &error);
-    REQUIRE(col);
-    CHECK(CBLCollection_Name(col) == kCBLDefaultCollectionName);
-    CHECK(CBLCollection_Count(col) == 0);
-    CBLCollection_Release(col);
+    REQUIRE(defaultCollection);
```

Comment:
You could do the required at a single place right after getting the default collection in the constructor.